src/cascadia/TerminalSettingsModel/MTSMSettings.h

```diff
@@ -24,8 +24,9 @@ Author(s):
     X(hstring, WordDelimiters, "wordDelimiters", DEFAULT_WORD_DELIMITERS)                                                                                                                             \
     X(bool, CopyOnSelect, "copyOnSelect", false)                                                                                                                                                      \
     X(bool, FocusFollowMouse, "focusFollowMouse", false)                                                                                                                                              \
-    X(bool, ForceFullRepaintRendering, "experimental.rendering.forceFullRepaint", false)                                                                                                              \
-    X(bool, SoftwareRendering, "experimental.rendering.software", false)                                                                                                                              \

```

Comment:
i think there are some folks that were using these in the past. Should we be migrating them to the roughly equivalent new settings?

Reply:
People who have been using `ForceFullRepaintRendering` likely have been doing something wrong or accidentally clicked it. `SoftwareRendering` did have its uses, but I believe that everyone who used it should try the new AtlasEngine again or migrate to using Direct2D. The exact choice here depends on their exact issues.